File: GoDeliverFrontEnd/src/app/Model/Route.ts
//Route Model
import { DateLogistics } from './DateLogistics';
import { Order } from './Order';
import { Deserializable } from './Deserializable';

export class Route implements Deserializable{ 
      public dateLogistics: DateLogistics;
      public newOrder: Order;

      deserialize(input: any) : this {
        Object.assign(this, input);
        this.dateLogistics= new DateLogistics().deserialize(input.dateLogistics);
        return this;
      }
  }
  

File: logistics-service/src/main/java/com/stackroute/logisticsservice/domain/Route.java
package com.stackroute.logisticsservice.domain;

public class Route {
	
	private DateLogistics dateLogistics;
	private Order newOrder;
	
	public Route() {
		
	}
	
	public Route(DateLogistics dateLogistics, Order newOrder) {
		super();
		this.dateLogistics = dateLogistics;
		this.newOrder = newOrder;
	}
	public DateLogistics getDateLogistics() {
		return dateLogistics;
	}
	public void setDateLogistics(DateLogistics dateLogistics) {
		this.dateLogistics = dateLogistics;
	}
	public Order getNewOrder() {
		return newOrder;
	}
	public void setNewOrder(Order newOrder) {
		this.newOrder = newOrder;
	}

	@Override
	public String toString() {
		return "Route [dateLogistics=" + dateLogistics + ", newOrder=" + newOrder + "]";
	}
	
	
}
